### ISSUE TITLE
NonControllingInterests and SignificantInvestmentsInSubsidiaries

### ISSUE THREAD

**Issue by mhoullier** [mhoullier/ESEF-block-tagging-hierarchy#16]

**Comment by mhoullier** [comment]
[Astrid Montagnier] The relation between these two concepts seems to be easily missed, but [IFRS 12.(10-12)](https://www.ifrs.org/issued-standards/list-of-standards/ifrs-12-disclosure-of-interests-in-other-entities.html/content/dam/ifrs/publications/html-standards/english/2022/issued/ifrs12/#IFRS12_10__IFRS12_P0071) clearly states that detail about material **non-controlling interests** is expected within the disclosure of **investments in subsidiaries**.

**Comment by mhoullier** [comment]
The expected contents described in [IFRS 12.12](IFRS12_12__IFRS12_P0088) seem to cover pretty much anything that could be disclosed about non-controlling interests; and disclosures about non-controlling interest in the reports do seem to always be present as a result of the application of IFRS 12. Assuming a hierarchical relation between these two seems safe to me (NonControllingInterests should always be multi-tagged with SignificantInvestmentsInSubsidiaries).

**Comment by Phamon** [comment]
I do not agree, for me, the disclosure of non-controlling interest corresponds to the non-controlling interest line items in the statements. IFRS12.12 means that the list of investments in subsidiaries which is expected in the disclosure of investments in subsidiaries must include the subsidiaries that are accounted for using the equity method (which produces the values for the captions minority interests in the financial statements): "subsidiaries with material non-controlling interests. 
![image](https://user-images.githubusercontent.com/43265216/190444679-d36a9fa2-5832-4de2-ba56-c0df4c08c00d.png)
